app/js/enhancedDate.js

```diff
@@ -1,5 +1,80 @@
-/* 
- * add your module definition for enhancedDate here
- * assign the module to a global variable
- * /
+var enhancedDate = (function iife() {
+    
+    // private functions
+    var publicAPI = {
+        setDate: function setDate(userDate) {
+            // Set working date to user input if date object given
+            if (userDate instanceof Date) {
+                var workingDate = userDate;
```

Comment:
You workingDate variable is local to the setDate scope, meaning no other methods will have access to it. Better would be to declare workingDate inside the module but outside of the setDate method. Additionally, objects are passed by reference in JavaScript when used as arguments. If you assign the object reference directly, and the object passed in is modified outside the module, it will modify the date stored inside your module as well, so better to use the passed-in value to instantiate a new Date object. Finally, the setDate method can take either a Date object or a millisecond value to use in setting a Date.